csrc/extensions.h

```diff
@@ -0,0 +1,9 @@
+
+#include <torch/torch.h>
+
+// for getpid()
+#ifdef _WIN32
```

Comment:
Why do we need those?

Reply:
For some reason, including `torch/python.h` leaks the `getpid()` headers. Those headers are not included in `torch/torch.h` so we had to manually add them as `getpid()` is used for random seed initialization in a few operators.

 Since `getpid()` is OS specific we have to include different headers on Unix and Windows.